components/typography/Heading.js

```diff
@@ -8,17 +8,38 @@ const factory = (type, defaultElement) => {
     static propTypes = {
       children: PropTypes.node,
       className: PropTypes.string,
+      color: PropTypes.oneOf(['white', 'neutral', 'mint', 'teal', 'violet', 'ruby', 'gold', 'aqua']),
       element: PropTypes.node,
+      soft: PropTypes.bool,
     };
 
     static defaultProps = {
       element: null,
+      soft: false,
```

Comment:
**PP** Prefer using `isCondition` signature for boolean values, eg. `isSoft`.

Reply:
As we recently discussed, we'd prefer to pass the props without the **is** prefix. But, functions that return a boolean value we should always prefix with **is**, **has**, ...

Reply:
+1, I prefer props without `is` as well, so you can do for example `<Heading soft>` (makes more sense than `<Heading isSoft>)`.

Reply:
Yeah, I was seriously confused, sorry boizzz 👍